.eslintrc.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
         "no-fallthrough": "warn", // TODO(bkendall): remove, allow to error.
         "no-unused-vars": "warn", // TODO(bkendall): remove, allow to error.
         camelcase: ["warn", { ignoreDestructuring: true }], // TODO(bkendall): remove, allow to error.
+        eqeqeq: ["error", "always", { null: "ignore" }],
```

Comment:
If this is only for `ts` files, this config should go in the block with `no-invalid-this`, grouping it with the other "permanent" rules we use. If this should be for _all_ files, put it in the first block under `rules`

Reply:
Done, moved to block with permanent rules.